bash: cannot set terminal process group (-1): Inappropriate ioctl for device
bash: no job control in this shell
 
:: initializing oneAPI environment ...
   bash: BASH_VERSION = 5.0.17(1)-release
   args: Using "$@" for setvars.sh arguments: --force
:: compiler -- latest
:: debugger -- latest
:: dev-utilities -- latest
:: dpl -- latest
:: inspector -- latest
:: itac -- latest
:: mpi -- latest
:: tbb -- latest
:: oneAPI environment initialized ::
 
Radii {'O': 1.35, 'Sr': 1.18, 'Ti': 0.42}
No closest distances between the ions were provided
Relying on Shannon radii only
Charges: {'O': -2.0, 'Sr': 2.0, 'Ti': 4.0}
[[-7.38665568  1.84610839  1.84610839  1.30539576  1.84610839  1.30539576
   1.30539576  1.06585118]
 [ 1.84610839 -7.38665568  1.30539576  1.84610839  1.30539576  1.84610839
   1.06585118  1.30539576]
 [ 1.84610839  1.30539576 -7.38665568  1.84610839  1.30539576  1.06585118
   1.84610839  1.30539576]
 [ 1.30539576  1.84610839  1.84610839 -7.38665568  1.06585118  1.30539576
   1.30539576  1.84610839]
 [ 1.84610839  1.30539576  1.30539576  1.06585118 -7.38665568  1.84610839
   1.84610839  1.30539576]
 [ 1.30539576  1.84610839  1.06585118  1.30539576  1.84610839 -7.38665568
   1.30539576  1.84610839]
 [ 1.30539576  1.06585118  1.84610839  1.30539576  1.84610839  1.30539576
  -7.38665568  1.84610839]
 [ 1.06585118  1.30539576  1.30539576  1.84610839  1.30539576  1.84610839
   1.84610839 -7.38665568]]
open_grid ~ generating matrix took 0.2795538902282715 seconds.
Set parameter WLSAccessID
Set parameter WLSSecret
Set parameter LicenseID to value 2625987
Traceback (most recent call last):
  File "/home/miru900/integer/brute_crystal/brute_crystal/central.py", line 69, in <module>
    selected_mod(key, settings)
  File "/home/miru900/integer/brute_crystal/brute_crystal/mods/mod_5.py", line 31, in mod_cell_int_periodic2
    position, op_energy = ip.integer(chem, ion_count, ewal_mat.dist, buck_info)
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/miru900/integer/brute_crystal/brute_crystal/cal_int/integer_program.py", line 18, in integer
    m = gb.Model() ; Vars = [[] for i in range(T)]
        ^^^^^^^^^^
  File "src/gurobipy/_model.pyx", line 146, in gurobipy._model.Model.__init__
  File "src/gurobipy/gurobi.pxi", line 60, in gurobipy._core.gurobi._getdefaultenv
  File "src/gurobipy/env.pxi", line 88, in gurobipy._core.Env.__init__
gurobipy._exception.GurobiError: Could not resolve host: token.gurobi.com (code 6, command POST https://token.gurobi.com/api/v1/tokens)
